Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -419,6 +480,16 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("GetOutlineSize")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddExpression(
+        "OutlineSize",
+        _("Outline size"),
+        _("Check the outline size in pixels."),
```

Comment:
This is an expression, so it's not "checking" anything :) 
To avoid losing time let's just copy paste the same thing as the name: "Outline size"